src/docfx/build/toc/TableOfContentsLoader.cs

```diff
@@ -59,6 +59,39 @@ internal class TableOfContentsLoader
             _joinTOCConfigs = config.JoinTOC.Where(x => x.ReferenceToc != null).ToDictionary(x => PathUtility.Normalize(x.ReferenceToc!));
         }
 
+        public enum TocHrefType
```

Comment:
suggest putting this `public enum` to a separate file

Reply:
Finished.